src/builtin.cpp

```diff
@@ -2451,7 +2453,7 @@ static int builtin_status(parser_t &parser, io_streams_t &streams, wchar_t **arg
     /// the non-flag subcommand form. While these flags are deprecated they must be supported at
     /// least until fish 3.0 and possibly longer to avoid breaking everyones config.fish and other
     /// scripts.
-    const wchar_t *short_options = L":cbilfnhj:t";
+    const wchar_t *short_options = L":bcfhij:lnut";
```

Comment:
How I should add the `current-function` parameter without messing up the deprecated style? Is there any sample on how to base my rewrite? With the symbol name is enough I will search it and look how it is done and apply on the PR changes.